files/default/kagent_utils/kagent_utils/kagent_config.py

```diff
@@ -97,6 +97,9 @@ def read_conf(self):
             else:
                 self.host_id = self.hostname
 
+            self.elk_key_file = self._config.get('agent', 'elk-key-file')
+            self.elk_certificate_file = self._config.get('agent', 'elk-certificate-file')
+            self.elk_cn = self._config.get('agent', 'elk-cn')  
```

Comment:
Can you also fix `test_kagent_config.py` and make sure it passes the tests?

Reply:
Fixed.